testing-guidance/workflow-tests/physics/physics-lkg-workflows.md

```diff
@@ -122,46 +192,108 @@ Any project that has the following Gems enabled:
-| Changing Rigid Body Properties in Workflows | <ol><li>In any of the workflows with a Rigid Body found on this page change the rigid body properties for the scenario under test.</li><li>Run the Simulation in Editor Game Mode or in the Game Launcher.</li></ol> | <ul><li>Scene behaves according to the the Ridgid Body changes made.</li><li>Extra care should be tested around the following Rigid Body Properties:</li><ul><li>Angular & Linear damping</li><li>COM</li><li>Kinematic Mode</li><li>Mass</li><li>Angular Velocity</li><li>Compute Intertia</li></ul></ul> |
+| Changing Rigid Body Properties in Workflows | <ol><li>In any of the workflows with a Rigid Body found on this page change the rigid body properties for the scenario under test.</li><li>Create a meshed Rigid body entity with each of the supported [PhysX Collider shape(s)](#physx-collider-shapes) or [PhysX Shape Collider shape(s)](#physx-shape-collider-shapes), position it above the terrain/any PhysX collider and change gravity and time steps values in System and Scene Configuration settings of PhysX Configuration (PREVIEW) tool .</li><li>Enable CCD in PhysX Configuration (PREVIEW) tool and set up a scene in which two or more CCD enabled PhysX rigid bodies collide with each other by using Initial linear velocities with values greater than 50.</li><li>Enable/Disable gravity enabled toggle in PhysX Rigid Body.</li><li>Enable/Disable start asleep toggle in PhysX Rigid Body.</li><li>Set sleep threshold for PhysX Rigid body and set up a scene in which PhysX Rigid body with set sleep treshold is impacted by other PhysX Rigid body.</li><li>Run the Simulation in the Editor Game Mode or in the Game Launcher after providing any of the changes mentioned above to PhysX Rigid body entity.</li></ol> | <ul><li>Scene behaves according to the changes made to PhysX Rigid Body entity.</li><li>Time step and gravity values set in PhysX Configuration (PREVIEW) tool correctly impact behaviors of rigid bodies in a scene.</li><li>CCD enabled Rigid bodies have better hit detection during scenarios involving big velocities.</li><li>Rigid bodies with gravity enabled toggle turned off do not react to gravity set in PhysX Configuration (PREVIEW)</li><li>Rigid bodies with start asleep toggle turned on have to be woken up by a force to start simulating.</li><li>Rigid bodies with set threshold go to sleep when applied forces are lower than set threshold.</li><li>Extra care should be tested around the following Rigid Body Properties:</li><ul><li>Angular & Linear damping</li><li>COM</li><li>Kinematic Mode</li><li>Mass</li><li>Angular Velocity</li><li>Compute Intertia</li></ul></ul> |
```

Comment:
suggestion: I'm not sure 'meshed rigid body' is super clear here, could this be clarified to 'A rigid body with a shape type of mesh' (something like that) or have 'meshed' defined elsewhere

Reply:
Indeed. In fact what I've meant was a PhysX Rigid Body entity with a mesh component added for visibility purposes.

I've created a new commit which reworded 'meshed rigid body' instances to 'PhysX Rigid body with entity with mesh added for visibility' to make it clearer.

Reply:
That sounds perfect thanks! Much appreciated 🙂